src/drivers/dw/dma.c

```diff
@@ -1411,6 +1437,56 @@ static int dw_dma_remove(struct dma *dma)
 	return 0;
 }
 
+static int dw_dma_in_data_size(struct dma_chan_data *chan)
+{
+	int size = 0;
```

Comment:
redundant initialisation. You could also move it two lines down and initialise with the correct value directly